README.md

```diff
@@ -32,55 +36,134 @@ Create `exams.sql` file with `INSERT INTO` query that adds entries to `exams` ta
 Write a query that outputs `year`, `modifier`, `student_cnt` columns that specify the number of students (`student_cnt`) in a class for all classes.
 
 ```sql
-PASTE YOUR CODE HERE
+WITH
+	aa AS (
+		SELECT year, modifier, id
+		FROM class
+		LEFT JOIN groups 
+		ON class.id = groups.class_id
+	)
+SELECT
+	 year,
+	 modifier,
+	 count(*) as student_cnt
+	 
+FROM aa
+GROUP BY 
+	year,
+	modifier;
 ```
 
 ### 2. Find the number of students for every year
 
 Write a query that outputs `year`, `student_cnt` columns that specify the number of students (`student_cnt`) in a class for each year (`year`).
 
 ```sql
-PASTE YOUR CODE HERE
+WITH
+	aa AS (
+		SELECT year, id
+		FROM class
+		LEFT JOIN groups 
+		ON class.id = groups.class_id
+	)
+SELECT
+	 year,
+	 count(*) as student_cnt
+	 
+FROM aa
+GROUP BY 
+	year;
 ```
 
 ### 3. Find unique subjects that students learn for a given year
 
 Write a query that outputs `name`, `surname`, `unique_lesson_cnt`, where `unique_lesson_cnt` is the number of **unique** lessons that a student identified by `name`, `surname` has.
 
 ```sql
-PASTE YOUR CODE HERE
+WITH
+	af AS (
+		SELECT *
+		FROM exams
+		JOIN students 
+		ON students.id = exams.student_id
+		JOIN lessons
+		ON lessons.id = exams.lesson_id
+	)
+SELECT
+	 name,
+	 surname,
+	 count(lesson_id) as unique_lesson_cnt
+	 
+FROM af
+GROUP BY 
+	name,
+	surname;
```

Comment:
Grouping by name and surname is not correct. What if there are namesakes (i.e. students with the same name and surnames, but different IDs)?